src/base/bittorrent/bencoderesumedatastorage.h

```diff
@@ -46,7 +46,7 @@ namespace BitTorrent
         Q_DISABLE_COPY(BencodeResumeDataStorage)
 
     public:
-        explicit BencodeResumeDataStorage(QObject *parent = nullptr);
+        explicit BencodeResumeDataStorage(const QString &path, QObject *parent = nullptr);
```

Comment:
maybe take `const QDir &path`

Reply:
I would keep it as `string` for consistency.